cmd/installer/cli/api.go

```diff
@@ -31,11 +31,15 @@ type apiConfig struct {
 	Logger          logrus.FieldLogger
 	MetricsReporter metrics.ReporterInterface
 	Password        string
+	TLSCertBytes    []byte
+	TLSKeyBytes     []byte
+	Hostname        string
 	ManagerPort     int
 	LicenseFile     string
 	AirgapBundle    string
-	ConfigChan      chan<- *apitypes.InstallationConfig
+	ConfigValues    string
 	ReleaseData     *release.ReleaseData
+	K0sOverrides    string
```

Comment:
this is a path and also of type ecv1beta1.Config so the name is probably not correct